document update (#25)
SAP-samples/cloud-extension-graph-sample: documentation/Set Up Mock Server/README.md

            }
        ]
```

> **Note** : Kinldy remove the "c4c" from  locatingPolicy/rules/name:"sap.graph.*"/local

Reviewer: Not a note, add this as a step as it needs to be executed every time.

8. In the `locating policy`, remove the `local` reference of `c4c` from the rule `sap.graph.*`. This will result in the rule as
```json
{
    "name": "sap.graph.*",
    "leading": "s4",
    "local": [
       
    ]
}
```

> After adding and removing, full configuration should look like this

Reviewer: ```suggestion
After the changes, full configuration should look like this
```
